clang/lib/AST/Interp/ByteCodeExprGen.cpp

```diff
@@ -810,9 +810,7 @@ bool ByteCodeExprGen<Emitter>::VisitArrayInitLoopExpr(
     const ArrayInitLoopExpr *E) {
   assert(Initializing);
   assert(!DiscardResult);
-  // TODO: This compiles to quite a lot of bytecode if the array is larger.
-  //   Investigate compiling this to a loop, or at least try to use
-  //   the AILE's Common expr.
```

Comment:
Removing the entire comment is not correct; it still compiles to a lot of code for larger arrays. Removing the AILE part of the comment is fine though.